### DISCUSSION TITLE
Possible to use on macOS?

### DISCUSSION THREAD

**Discussion by bbrendon** [kennetek/gridfinity-rebuilt-openscad#146] [Q&A]
I'm only an openscad user because of this project. I installed the nightly build today on an M1 and after opening the file gridfinity-rebuilt-lite.scad, the program is basically unusable.  I then enabled fast-csg and the same results.

Am I wasting my time messing with this on macOS?  If not, then how the hell do you get this thing working?

**Comment by skandragon** [comment]
I render this on my Mac, and it "just works" but I may have installed a development version of OpenSCAD.  I also have an M1 based Mac.

**Comment by skandragon** [comment]
I have OpenSCAD 2021.01 installed, apparently.  It renders fast enough but it can take several seconds before some of the more detailed boxes show up.  It seems slow, but it's a generate-once-print-many situation generally.

**Comment by skandragon** [comment]
I also tried with the latest scad development version, and it also works, but is slow.  I kinda wonder if I (or someone) should build a STL generator for these online...)

**Reply by Ruudjhuu** [reply]
What do you mean with STL generator. OpenScad is basically an STL generator.

**Reply by skandragon** [reply]
What I mean is, I might build a "design your box" web site that just sends back STLs, so people can get the benefit from this stuff without having to install OpenSCAD or fight with slow generation times.
